chirp/drivers/ft2800.py

```diff
@@ -182,6 +189,17 @@ class FT2800Radio(YaesuCloneModeRadio):
     _block_sizes = [8, 7680]
     _memsize = 7680
 
+    @property
+    def subtype(self):
+        if 'subtype_idblock' in self.metadata:
+            return bytes(self.metadata['subtype_idblock'], 'latin1')
+        else:
+            return SUPPORTED_IDBLOCKS[0]
+
+    @subtype.setter
+    def subtype(self, value):
+        self.metadata = {'subtype_idblock': value.decode('latin1')}
```

Comment:
This encode and decode from latin1 is not quite right. I'm guessing you're doing this because the load and save doesn't restore the bytes because we're using json for the payload?

In that case, how about we store this as an integer array of byte values that will store properly in JSON? That will make the getter/setter you have here even more useful. So something like this for the setter:
```
self.metadata = {'subtype_idblock': [x for x in value]}
```
and this for the getter:
```
return bytes(self.metadata['subtype_idblock'])
```

Reply:
Agree, the latin1 thing was stupid.